elvis.config

```diff
@@ -26,6 +26,11 @@
                     #{regex => "^([a-z][a-z0-9]*_?)*(_SUITE)?$",
                       ignore => []}
                   },
+                  {
+                    elvis_style,
+                    variable_naming_convention,
+                    #{regex => "^([A-Z][0-9a-zA-Z]*)*$", ignore => []}
```

Comment:
I don't think you need the external star, i.e. you can get the same results with just `[A-Z][0-9a-zA-Z]*`.
I'm also pretty sure your regex will render `_ThisTotallyValidVariable` as invalid.
I think the proper default value should be along the lines of `^_?[A-Z][0-9a-zA-Z]*$` or maybe better we should ignore the leading `_` when matching regexes in code.